### PR TITLE
bentobox 0.7.7 (new cask)

### PR DESCRIPTION
**Pull request by shayne**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/b/bentobox.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  name "BentoBox"
+  desc "Window manager that organizes desktop applications into predefined zones"
+  homepage "https://bentoboxapp.com/"
+
+  livecheck do
+    url "https://releases.bentobox.friendlyventures.org/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  depends_on macos: ">= :ventura"
```
Comment (khipp) [inline Casks/b/bentobox.rb:16]:
```suggestion
  auto_updates true
  depends_on macos: ">= :ventura"
```

The app can install updates on its own.

File: Casks/b/bentobox.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  app "BentoBox.app"
+
+  zap trash: [
+    "~/Library/Preferences/org.friendlyventures.BentoBox.plist",
+    "~/Library/Saved Application State/org.friendlyventures.BentoBox.savedState",
```
Comment (khipp) [inline Casks/b/bentobox.rb:22]:
```suggestion
    "~/Library/Application Support/org.friendlyventures.BentoBox",
    "~/Library/Caches/org.friendlyventures.BentoBox",
    "~/Library/HTTPStorages/org.friendlyventures.BentoBox",
    "~/Library/Preferences/org.friendlyventures.BentoBox.plist",
    "~/Library/Saved Application State/org.friendlyventures.BentoBox.savedState",
    "~/Library/WebKit/org.friendlyventures.BentoBox",
```

**Review by khipp — COMMENTED** [review]
Thank you for your first contribution to `homebrew-cask`, @shayne! 🎉

I left a few minor suggestions below.

File: Casks/b/bentobox.rb (hunk 1 above)
Comment (shayne) [reply Casks/b/bentobox.rb:17]: Added!

File: Casks/b/bentobox.rb (hunk 2 above)
Comment (shayne) [reply Casks/b/bentobox.rb:26]: Thanks for the tip, added!

**Review by khipp — APPROVED** [review]
Thank you!

**Pull request merged**